third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -593,6 +593,29 @@ convertFp32ToFp16RTZ(Location loc, ConversionPatternRewriter &rewriter,
   return ret;
 }
 
+// Fp32->Fp16 (RTNE) in GFX950
+static SmallVector<Value>
+convertPkFp32ToFp16(Location loc, ConversionPatternRewriter &rewriter,
```

Comment:
We have another utility function `cvtFp32ToFp16RTNE` in `Utility.cpp` and also I think `fp_to_fp` can do similiar conversion from fp32 to fp16. Can we unify them?